api/SpeechRecognitionErrorEvent.json

```diff
@@ -51,35 +39,76 @@
-              "notes": "You'll need to serve your code through a web server for recognition to work."
-            },
-            {
-              "alternative_name": "webkitSpeechRecognitionError",
-              "version_added": "2.0",
-              "version_removed": "12.0"
-            }
-          ],
-          "webview_android": [
-            {
-              "version_added": "77",
-              "notes": "You'll need to serve your code through a web server for recognition to work."
-            },
-            {
-              "alternative_name": "webkitSpeechRecognitionError",
-              "version_added": "≤37",
-              "version_removed": "77"
-            }
-          ]
+          "samsunginternet_android": {
+            "alternative_name": "webkitSpeechRecognitionError",
+            "version_added": "2.0",
+            "notes": "You'll need to serve your code through a web server for recognition to work."
+          },
+          "webview_android": {
+            "alternative_name": "webkitSpeechRecognitionError",
+            "version_added": "≤37",
+            "notes": "You'll need to serve your code through a web server for recognition to work."
+          }
         },
         "status": {
           "experimental": true,
           "standard_track": true,
           "deprecated": false
         }
       },
+      "SpeechRecognitionErrorEvent": {
+        "__compat": {
+          "description": "<code>SpeechRecognitionErrorEvent()</code> constructor",
+          "spec_url": "https://wicg.github.io/speech-api/#dom-speechrecognitionerrorevent-speechrecognitionerrorevent",
+          "support": {
+            "chrome": {
+              "alternative_name": "webkitSpeechRecognitionError",
+              "version_added": "33"
+            },
+            "chrome_android": {
+              "alternative_name": "webkitSpeechRecognitionError",
+              "version_added": "33"
+            },
+            "edge": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": "14.1"
+            },
+            "safari_ios": {
+              "version_added": "14.5"
+            },
+            "samsunginternet_android": {
+              "alternative_name": "webkitSpeechRecognitionError",
+              "version_added": "2.0"
+            },
+            "webview_android": {
+              "alternative_name": "webkitSpeechRecognitionError",
+              "version_added": "≤37"
+            }
+          },
+          "status": {
+            "experimental": true,
```

Comment:
This matches the parent feature, so alright then. It can be fixed together separately.